clar2wasm/src/words/responses.rs

```diff
@@ -71,16 +71,12 @@ impl Word for IsErr {
         _expr: &SymbolicExpression,
         args: &[SymbolicExpression],
     ) -> Result<(), GeneratorError> {
-        match traverse_response(generator, builder, args) {
-            Ok(_) => {
-                // Add one to stack
-                // and proceed with a XOR operation
-                // to invert the indicator value
-                builder.i32_const(1);
-                builder.binop(BinaryOp::I32Xor);
-            }
-            Err(e) => return Err(e),
-        };
+        let _ = traverse_response(generator, builder, args);
```

Comment:
```suggestion
        traverse_response(generator, builder, args)?;
```

I think this is what you're looking for.

Reply:
Resolving this one since it was already brought up in another discussion.

Reply:
This is wrong, you are ignoring the potential error.
Why not just committing my suggestion?

Reply:
No, I'm agreeing with your suggestion. I just hadn't realized you had already suggested it when I opened this discussion, so I closed this one.

Reply:
ah sorry, I was speaking to Chris, no idea why github added my comment to yours.
I guess it merges them when they come from the same location?

Reply:
Sorry, I missed the question mark early returning an error, if need. 👍